motoman_driver/MotoPlus/mpMain.c

```diff
@@ -57,6 +57,9 @@ void mpUsrRoot(int arg1, int arg2, int arg3, int arg4, int arg5, int arg6, int a
 #ifdef DX100
 	Ros_Sleep(10000);  // 10 sec. delay to enable DX100 system to complete initialization
 #endif
+#ifdef DEBUG
+	Ros_Debug_Init();

```

Comment:
This is not actually needed. This function is called in `Debug_BroadcastMsg`

Reply:
OK, removed